### ISSUE TITLE
unable to install

### ISSUE THREAD

**Issue by lw1582** [ngzhian/pycrunchbase#11]
Hi,

I'm trying to install pycrunchbase on Python 2.7 and I keep getting an error message. Do you know how to fix this? Thanks!

-----------------------------------------------------------------------------------------

Processing dependencies for pycrunchbase==0.3.0
Searching for six==1.9.0
Reading https://pypi.python.org/simple/six/
Download error on https://pypi.python.org/simple/six/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:581) -- Some packages may not be found!
Couldn't retrieve index page for 'six'
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
Download error on https://pypi.python.org/simple/: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:581) -- Some packages may not be found!
No local packages or download links found for six==1.9.0
error: Could not find suitable distribution for Requirement.parse('six==1.9.0')

**Comment by ngzhian** [comment]
Hm, not too sure about this, a quick search turned out some results that might help:

http://stackoverflow.com/questions/25981703/pip-install-fails-with-connection-error-ssl-certificate-verify-failed-certi

http://stackoverflow.com/questions/16094187/pip-3-3-ssl-verification-error-mac-osx-10-5-8

try if those work for you?

**Comment by lw1582** [comment]
Thanks for the pointeres! I will give it a try.

Lucy

**Comment by ngzhian** [comment]
Hope it works, do leave a message and tell us how it goes :)

**Comment by lw1582** [comment]
so it's a little weird...even though i get that error in installation, the
module seems to work when i import to python...i am getting a different
kind of "error" where i get a "no access" message from crunchbase when i
try to make certain calls. I emailed crunchbase about this but haven't
heard back yet. I'm not sure if this is from the package not installing
correctly on my computer or an issue with my API key...

The things I tried to retrieve are things like founders or competitors. I
also get returned a total number, but I can't see the actual founders or
competitors of any company I look up...not sure if you are the right person
to help out with this but I'll keep you posted if I figure it out or hear
back from CB.

Thanks for the follow up!

Lucy
